pkg/apis/core/v1/helper/helpers.go

```diff
@@ -92,10 +92,14 @@ func IsOvercommitAllowed(name v1.ResourceName) bool {
 		!IsHugePageResourceName(name)
 }
 
+func IsStorageLimitResourceName(name v1.ResourceName) bool {
```

Comment:
I would rename this to `IsVolumeLimitResourceName` to be consistent with the rest of the code.